xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -5278,6 +5278,11 @@ xe.admin.users.filter.firstname=First name filter
 xe.admin.users.lastname=Last name
 xe.admin.users.filter.lastname=Last name filter
 
+#######################################
+## until 10.11RC1
+#######################################
+core.viewers.history.empty="The history of this page is empty."
```

Comment:
`## until *`` comments as used for deprecated entries (when we stop using them)

Reply:
Right I should have known that. Anyway it's fixed.